gtk3/sample/tutorial/README.md

```diff
@@ -435,6 +435,88 @@ https://developer.gnome.org/gtk3/stable/ch01s04.html#id-1.2.3.12.7
 
 *    exampleapp3/exampleapp.rb
 
+In this step, we make our application show the content of all the files that it is given on the commandline.
+To this end, we need to easily access widgets from the template. So we use the method `Gtk::Widget#bind_template_child` like in the code below:
```

Comment:
`Gtk::Widget.bind_template_child` （`.` not `#`）is better because `.` means that the method is a singleton (class) method not an instance method.